client/extensions/woocommerce/state/ui/payments/methods/test/reducer.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import reducer, { initialState } from '../reducer';
```

Comment:
It would be better to define the initialState inside the test or in a fixtures file, otherwise you're testing "does a==a"